src/actions/editProfile.action.js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios';
+import toastr from 'toastr';
+
+import config from '../config';
+import { EDIT_PROFILE_BEGINS, EDIT_PROFILE_SUCCESS, EDIT_PROFILE_FAILURE } from './types';
+
+export const editProfileBegins = () => ({
+  type: EDIT_PROFILE_BEGINS
+});
+
+export const editProfileSuccess = user => ({
+  type: EDIT_PROFILE_SUCCESS,
+  user
+});
+
+export const editProfileFailure = error => ({
+  type: EDIT_PROFILE_FAILURE,
+  error
+});
+
+const editProfile = user => (dispatch) => {
```

Comment:
`const editProfile = ({ user }) => (dispatch) => {`
spo you avoid the `user.user`